vyper/parser/parser.py

```diff
@@ -149,13 +149,13 @@ def mk_method_identifiers(code):
     return o
 
 
-def parse_events(sigs, _events, custom_units=None):
+def parse_events(sigs, _events, custom_units=None, custom_structs={}):
```

Comment:
Use `custom_structs=None` and `if None: custom_structs = {}` in function body here instead (default args as such can cause a lot of trouble).